xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -32,8 +32,11 @@ use xmtp_proto::{
 async fn create_tls_channel(address: String) -> Result<Channel, Error> {
     let channel = Channel::from_shared(address)
         .map_err(|e| Error::new(ErrorKind::SetupError).with(e))?
+        .initial_connection_window_size(Some((1 << 31) - 1))
```

Comment:
**Purpose:** This setting controls the size of the initial connection-level flow control window for HTTP/2, which is the underlying protocol for gRPC.

**Functionality:** Flow control in HTTP/2 manages how much data can be in flight on the network. Setting the initial connection window size to (1 << 31) - 1 (the maximum possible value for a 32-bit integer, which is 2,147,483,647 bytes) essentially allows the client to receive a very large amount of data from the server before needing to acknowledge receipt and permit more data to be sent. This can be particularly useful in high-latency networks or when transferring large amounts of data.

**Impact:** Increasing the window size can improve throughput by allowing more data to be in transit at a time, but it may also increase memory usage and can potentially lead to inefficient use of bandwidth if the network is unreliable.

Reply:
@neekolas any strong feelings for or against this one? This one feels like the biggest change but should help with latency issues.

Reply:
This seems worth a shot. I'm not quite sure how it'll work with GRPC streams, but it's a good sign that The Graph has been using this setting in production and it hasn't blown up.

Reply:
Based on my read of that code comment, it'll only come into play for clients that are connecting to a _lot_ of different streams at once. But it's worth a try.